pyttsx3/drivers/avsynth.py

```diff
@@ -0,0 +1,143 @@
+import objc
+from AppKit import NSSpeechSynthesizer
+from Foundation import NSObject, NSTimer, NSRunLoop, NSDefaultRunLoopMode, NSURL, NSDate
+from PyObjCTools import AppHelper
+from ..voice import Voice
+
+
+# noinspection PyPep8Naming
+def buildDriver(proxy):
+    driver = AVSpeechDriver.alloc().init()
+    driver.setProxy(proxy)
+    driver.initialize_busy_state()
+    return driver
+
+
+class AVSpeechDriver(NSObject):
+    def init(self):
+        self = objc.super(AVSpeechDriver, self).init()
+        if self is None:
+            return None
```

Comment:
% `pipx run ruff rule PLE0101`
https://docs.astral.sh/ruff/rules/return-in-init

```suggestion
            return
            # or perhaps better...
            raise RuntimeError("Unable to instantiate an AVSpeechDriver")
```